test/Microsoft.AspNetCore.SignalR.Common.Tests/Internal/Protocol/TestHubMessageEqualityComparer.cs

```diff
@@ -47,6 +47,11 @@ private bool InvocationMessagesEqual(HubMessage x, HubMessage y)
                 left.NonBlocking == right.NonBlocking;
         }
 
+        private bool CancelInvocationMessagesEqual(HubMessage x, HubMessage y)
+        {
+            return x is CancelInvocationMessage && y is CancelInvocationMessage;
```

Comment:
Shouldn't this check the invocation ids are the same?

Reply:
`invocationId` is common to `HubMessage` and is being checked in `Equals`.